components/rmrk/Gallery/History.vue

```diff
@@ -226,16 +265,15 @@ export default class History extends mixins(ChainMixin, KeyboardEventsMixin) {
     this.$router
       .replace({
         path: String(this.$route.path),
-        query: { ...this.$route.query, [key]: encodeURIComponent(value) },
+        query: { ...this.$route.query, [key]: value },
       })
       .catch(this.$consola.warn /*Navigation Duplicate err fix later */)
   }
 
   protected createTable(): void {
     this.data = []
     this.copyTableData = []
-    const priceCollectorMap: Record<string, string> = {}
-    const ownerCollectorMap: Record<string, string> = {}
+    let curPrice
```

Comment:
Every time someone uses `let`
vikiival gets sad

Reply:
Why gets sad for it  0、0

`let` and `const` are both useful.

`let` is used for some changeable value like this `curPrice`